### PR TITLE
Solace Integration Docs

### PR DESCRIPTION
**Pull request by ruthryi**

### CODE REVIEW

File: en/docs/tutorials/integrating-with-solace.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        service_url = <Base URL of the Solace broker>
+        username = <Username of the Solace broker account>
+        password = <Password of the Solace broker account>
+        ws_endpoint = <WS Endpoint of the Solace broker environment>
+        wss_endpoint = <WSS Endpoint of the Solace broker environment>
+        http_endpoint = <HTTP Endpoint of the Solace broker environment>
+        https_endpoint = <HTTPS Endpoint of the Solace broker environment>
+        show_as_token_endpoint_url = false
+
+        [apim.gateway.environment.properties]
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:33]: @ruthryi Do we need to mention that these additional properties are not optional and those are mandatory?

File: en/docs/tutorials/integrating-with-solace.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Before you begin: Make sure that you have integrated the Solace event broker with WSO2 API Manager as shown in the above section.
+
+1. Login to the API Publisher via **https://&lt;hostname>:9443/publisher**
+2. Click **CREATE API** and then click **Import AsyncAPI Definition**
+    <a href="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png"><img src="{{base_path}}/assets/img/tutorials/solace/import-asyncapi.png" alt="Import AsyncAPI Definition" name="Import AsyncAPI Definition"></a>
+
+3. The following two options to import the AsynAPI definition appear.
+    1. AsyncAPI URL - If you select this option, you need to provide a URL of a Solace API Specification.
+    2. AsyncAPI File - If you select this option, click Browse File to Upload and upload a file, which contains a Solace AsyncAPI definition.
+    Select your preferred method and click **NEXT**.
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:56]: Shall we add the following note here?

"Note - **_If the Async API specification is a valid Solace Async API specification, the portal will display the “Identified as Solace Event Portal API” tag._**"

File: en/docs/tutorials/integrating-with-solace.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+4. Edit the Solace Async API information and click **Create**.
+
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:59]: Shall we add the screenshot to this step to show the Create option with identified solace protocols?

File: en/docs/tutorials/integrating-with-solace.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+The topics will be created automatically from the Solace AsyncAPI definition provided in the previous step.
+
+### Step 3 - Viewing the Async API definition
+
+1. Go to **API Configurations** and click **AsyncAPI Definition** to see the definition of the API you created in the previous step. 
+    <a href="{{base_path}}/assets/img/tutorials/solace/asyncapi-definition.png"><img src="{{base_path}}/assets/img/tutorials/solace/asyncapi-definition.png" alt="AsyncAPI Definition" name="AsyncAPI Definition"></a>
+
+    !!! note
+        The Solace API specification cannot be edited from the API manager Publisher portal. But the specification can be downloaded and changed externally to make the required changes. 
+
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:77]: Shall we add this part saying at the moment we only support one subscription policy option?

Suggestion: **_Warning_ - The current version of WSO2 API Manager only supports AsyncUnlimited Protocol as the API level subscription policy for Solace APIs. Once a subscription is made, that policy will be added automatically.**

File: en/docs/tutorials/integrating-with-solace.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
+# External Broker and Gateway Integration with API Manager
+
+WSO2 API Manager now supports integrating external third party Gateways and Brokers. Solace is an advanced event broker which is efficiently streaming events and information across cloud, on-premises, and IoT environments. In this section, you will integrate the Solace gateway environment with API Manager.
+
+## Integrating the Solace Event Broker
+
+1. Go to the [Solace PubHub+](https://solace.com/products/event-broker/) portal.
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:7]: To set up a solace PubHub+ account they have this Getting started guide. If its helpful and required we can add this here.

[1]. https://docs.solace.com/Cloud/ggs_signup.htm

File: en/docs/tutorials/integrating-with-solace.md (hunk 1 above)
Comment (ruthryi) [reply en/docs/tutorials/integrating-with-solace.md:33]: I think that info should go into the Config catalog. Can you pls provide which ones are mandatory?

Comment (Chamindu36) [reply en/docs/tutorials/integrating-with-solace.md:33]: The gateway environment configs are already in the docs[1].

[1]https://apim.docs.wso2.com/en/latest/reference/config-catalog/#virtual-host-of-api-m-gateway-environment-configurations

File: en/docs/tutorials/integrating-with-solace.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+        username = <Username of the Solace broker account>
+        password = <Password of the Solace broker account>
+        ws_endpoint = <WS Endpoint of the Solace broker environment>
+        wss_endpoint = <WSS Endpoint of the Solace broker environment>
+        http_endpoint = <HTTP Endpoint of the Solace broker environment>
+        https_endpoint = <HTTPS Endpoint of the Solace broker environment>
+        show_as_token_endpoint_url = false
+
+        [apim.gateway.environment.properties]
+        Organization = <Password of the Solace broker account>
```
Comment (Chamindu36) [inline en/docs/tutorials/integrating-with-solace.md:38]: Shall we add a note saying these properties are not optional?

File: en/docs/tutorials/integrating-with-solace.md (hunk 3 above)
Comment (Chamindu36) [reply en/docs/tutorials/integrating-with-solace.md:66]: @ruthryi Shall we add the ss for this?

**Review by Chamindu36 — APPROVED** [review]

**Pull request merged**